src/Razor/src/rzls/Program.cs

```diff
@@ -20,7 +20,7 @@ public static async Task Main(string[] args)
         {
             if (args[i].Contains("debug", StringComparison.OrdinalIgnoreCase))
             {
-                await Console.Error.WriteLineAsync($"Server started with process ID {Environment.ProcessId}");
+                await Console.Error.WriteLineAsync($"Server started with process ID {Environment.ProcessId}").ConfigureAwait(false);
```

Comment:
`ConfigureAwait(false)` isn't necessary in `rzls`. These usages are gratuitous.

Reply:
I agree this is unnecessary, but just to confirm my understanding, it doesn't cause actual harm, right? The language server itself will utilize the thread pool, so it's not like this is the only thing thats requiring it and therefore we're paying an extra cost.

There are other `.ConfigureAwait(false)`s already in this file, I'm happy to clean them up (change to `true`) if its better, my main motivation for the PR was to get to a consistent state in the language server itself. I'm also fine just enabling the analyzer in that project instead.

Reply:
Nope, it doesn't do any harm.